docs-src/descriptions/multiple-sources.jade

```diff
@@ -82,18 +82,18 @@ p.
   lined up with each other. For example if you have two sources with values
   #[tt [1, 2, 3]] and #[tt [4, 5, 6, 7]], the result stream will emit
   #[tt [1, 4]], #[tt [2, 5]], and #[tt [3, 6]].
-  The result stream will emit next value only when it has at least one value
-  from each of sources.
+  The result stream will emit the next value only when it has at least one value
+  from each source.
 
 p.
-  You can also provide a #[b combinator] function, in this case instead
-  of emitting array of values, they will be passed to #[b combinator] as arguments,
-  and returned value will be emitted (same as in #[a(href="#combine") combine],
+  You can also provide a #[b combinator] function. In this case, instead
+  of emitting an array of values, they will be passed to #[b combinator] as arguments,
+  and the returned value will be emitted (same as in #[a(href="#combine") combine],
   #[a(href="#sampled-by") sampledBy], etc.)
 
 p.
   Also in #[b zip] you can pass ordinary arrays along with observables
-  in #[b sources], e.g. #[tt Kefir.zip([obs,&nbsp;[1,&nbsp;2,&nbsp;3]],&nbsp;fn)].
+  in the #[b sources], e.g. #[tt Kefir.zip([obs, [1, 2, 3]], fn)].
   In other words, #[b sources] is an array of observables and arrays,
```

Comment:
`&nbsp;` here are intentional, because code looks ugly when splitted to 2 lines.